official-antistasi-community/A3-Antistasi
# Corrected CfgSounds. Moved from maps to core.

## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [ ] Enhancement

### What have you changed and why?
CfgSounds was defined in Maps. Normally, this wouldn't be a problem however, the engine functionality has depreciated recently and using @ to call sounds from other mods no longer works.

https://community.bistudio.com/wiki/Description.ext#CfgSounds

### Please specify which Issue this PR Resolves.
N/A

### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

1. Start Mission
2. Debug: {player playSound "Sound Class from CfgSounds"} forEach allPlayers;
3. Observe sound is being played correctly

********************************************************
Notes:
